core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -230,6 +230,9 @@ class SnappySession(@transient private val sc: SparkContext,
       case LogicalRelation(br, _, _) =>
         br match {
           case d: DestroyRelation => d.truncate()
+          case _ => if (!ignoreIfUnsupported) {
```

Comment:
Its for cleanup of relations which supports truncate. It was causing a scala match error if we dont handle it.